src/window.cpp

```diff
@@ -3516,8 +3516,9 @@ void ChangeVehicleViewports(VehicleID from_index, VehicleID to_index)
  */
 void RelocateAllWindows(int neww, int newh)
 {
-	Window *w;
+	DeleteWindowById(WC_DROPDOWN_MENU, 0);
```

Comment:
![resizedropdown](https://user-images.githubusercontent.com/3152990/51139864-4cb12a80-1845-11e9-83b1-8b03c28636b4.png)
Should I open another PR for this?

Reply:
Nah, is fine here

Reply:
@LordAro 
I saw your comment about:
> i feel like max\<uint\> would be cleaner - no need to cast after the function call (+ available_height_above below)


... but cannot find it now on GitHub.
I think the parameters of the max function can be negative. I prefer making the cast after the max is calculated (it won't be negative). Is it really necessary to use max\<uint\> ?

Reply:
I deleted it because I saw it was stupid, don't worry about it :)